modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -1362,6 +1364,26 @@ private void _validateName(
 		}
 	}
 
+	private void _validateObjectDefinitionStorageType(
```

Comment:
Better call this method `_validateBusinessType` and then you can put the call to `_validateBusinessTypeEncrypted` inside it.